### ISSUE TITLE
[Features Request] Measurement function

### ISSUE THREAD

**Issue by BambuLabDiscord** [bambulab/BambuStudio#3145]
Would like the measurement function in Bambu Studio.  It will be useful when we scale an object, we can measure specific parts. At the moment need to switch to another application to check the edited scale.

**Comment by xSepticSidx** [comment]
TBH I'm surprised that we do not already have this, the ability to measure edge to edge, corner/radius etc. are essential in CAD.
Taking the model outside of Bambu Studio just for verification means that Studio is lacking so I fully support this request as hard as it may be to implement this new feature.

**Comment by RaphTronic** [comment]
Indeed, the lack of a measurement tool makes the workflow very cumbersome when scalig parts, when adjusting clearances for parts that joint together, and when double checking a model's correct version / dimensions before printing.
